barsys/templates/email_dutch/bank_account_details.html.html

```diff
@@ -17,5 +17,11 @@
         <th>Betalingskenmerk</th>
         <td>{{ pybarsys_preferences.EMAIL.BANK_ACCOUNT_PAYMENT_REFERENCE }} {{ recipient.display_name|clean_str }}</td>
     </tr>
+    {% if invoice.comment != "" %}
+    <tr>
+        <th>Comment</th>
```

Comment:
Can you adapt the English and German mail templates, too? It's probably easier for you to do it if there will be more small changes (e.g. the name of  `urlcheck`)..

Dutch: Shouldn't it be `Commentaar`?
English: `Comment`
German: `Kommentar`

Reply:
yeah ur right bad habbit to do everything in english